pootle/apps/pootle_format/default.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+
+POOTLE_FORMATS = [
+    ("po",
+     dict(title='Gettext PO file',
+          extension="po",
+          template_extension="pot")),
+    ("xliff",
+     dict(title='XLIFF Translation File',
+          extension="xliff",
+          template_extension="xliff")),
+    ("xlf",
+     dict(title='XLIFF Translation File',
```

Comment:
i can add it further up the stack - probs on `__str__` is a good idea actually - but for ref - here is where its actually used - https://github.com/translate/pootle/pull/4963/files#diff-83014e45acc5cd6424075fd6a8861be1R37